demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -189,6 +193,21 @@ def _is_pack_meta_file_structure_valid(self):
 
         return True
 
+    def _is_valid_contributor_pack_support_details(self):
+        """Checks if email or url exist in contributed pack details."""
+        try:
+            pack_meta_file_content = json.loads(self._read_file_content(self.pack_meta_file))
+            if pack_meta_file_content[PACK_METADATA_SUPPORT] in CONTRIBUTORS_LIST:
+                if not pack_meta_file_content[PACK_METADATA_URL] and not pack_meta_file_content[PACK_METADATA_EMAIL]:
+                    if self._add_error(Errors.pack_metadata_missing_url_and_email(), self.pack_meta_file):
+                        return False
+
+        except (ValueError, TypeError):
+            if self._add_error(Errors.pack_metadata_isnt_json(self.pack_meta_file), self.pack_meta_file):
```

Comment:
Maybe save `Errors.pack_metadata_isnt_json(self.pack_meta_file), self.pack_meta_file` as a variable at the beggining of the method

Reply:
no need, this is the way this whole file is written